starter-code/js/index.js

```diff
@@ -1,52 +1,87 @@
-function deleteItem(e){
-
-}
-
-function getPriceByProduct(itemNode){
-
+let pu = document.getElementsByClassName("pu");
+let productQuantity = document.getElementsByTagName("input");
+let subtot = document.getElementsByClassName("subtot");
+let productRow = document.getElementsByClassName("product");
+let tbody = document.querySelector(".table");
+
+function deleteItem(e) {
+  console.log(e.target);
+  e.target.parentElement.parentElement.remove();
+  // for (var i = 0; i < productRow.length; i++) {
+  //   tbody.removeChild(productRow[i]);
+  // }
```

Comment:
If you wanted to update the total price when you delete an element, you could add :
  `getTotalPrice();`
At the end of your delete function : )